src/sentry/services/hybrid_cloud/auth/impl.py

```diff
@@ -296,18 +292,10 @@ def __init__(self, req: AuthenticationRequest) -> None:
             _auth_user_hash=req.user_hash,
             _nonce=req.nonce,
         )
-        self.POST = FakeRequestDict(
-            _sentry_request_signature=req.signature,
```

Comment:
@corps We won't need `find_signature()` anymore if we're n longer setting `_sentry_request_signature` and `_`: https://github.com/getsentry/sentry/blob/5aa9698190e1125e2e808f995c232a93781006db/src/sentry/utils/linksign.py#L42-L43

Reply:
We will, because we do, but we won't route it through the control silo because it can be resolved locally more easily.

Reply:
@corps Actually I take that back. We still need `find_signature()` for the unsubscribe links.

But I have no idea where we set `_sentry_request_signature` anywhere in the codebase for both `sentry` and `getsentry`. 

I do know where we set `_` tho.